.circleci/config.yml

```diff
@@ -111,6 +111,20 @@ commands:
           paths:
             - "*.results"
 
+  run_test:
+    description: "Run tests matching a pattern"
+    parameters:
+      pattern:
+        type: string
+        default: ""
+    steps:
+      - checkout
+      - run: pip3 install virtualenv  # TODO: this can be baked into the image
```

Comment:
Though the image is not updated that often, so it might be better to have the latest `virtualenv` deployed each time.